java/animate-3d-graphic/README.md

```diff
@@ -1,58 +1,52 @@
-# Animate 3D Graphic
+# Animate 3D graphic
 
-Animate a graphic's position and orientation and follow it with the camera.
+An `OrbitGeoElementCameraController` follows a graphic while the graphic's position and rotation are animated.
 
-![Animate 3D Graphic App](animate-3d-graphic.png)
+![Image of animate 3D graphic](animate-3d-graphic.png)
 
 ## How to use the sample
 
 Animation Controls (Top Left Corner):
+
 * Select a mission -- selects a location with a route for plane to fly.
 * Mission progress -- shows how far along the route the plane is. Slide to change keyframe in animation.
 * Play -- toggles playing and stopping the animation.
```

Comment:
When the sample is running,the Play button says Stop (changes when you hit it), so let's make that a bit clearer here
```suggestion
* Play/Stop -- toggles playing and stopping the animation.
```